Formula/clang-format.rb

```diff
@@ -47,31 +46,44 @@ def install
     if build.head?
       ln_s buildpath/"clang", buildpath/"llvm/tools/clang"
     else
-      (buildpath/"tools/clang").install resource("clang")
+      resource("clang").stage do |r|
+        (buildpath/"tools/clang").install Pathname("clang-#{r.version}.src").children
+      end
     end
 
-    llvmpath = build.head? ? buildpath/"llvm" : buildpath
+    llvmpath = build.head? ? buildpath/"llvm" : buildpath/"llvm-#{version}.src"
 
     mkdir llvmpath/"build" do
       args = std_cmake_args
-      args << "-DLLVM_EXTERNAL_PROJECTS=\"clang\""
+      args << "-DLLVM_EXTERNAL_PROJECTS=clang"
+      args << "-DLLVM_INCLUDE_BENCHMARKS=OFF"
       args << ".."
-      system "cmake", "-G", "Ninja", *args
-      system "ninja", "clang-format"
+      system "cmake", *args
+      system "make", "clang-format"
```

Comment:
Better use `"cmake", "--build", ".", "--target", "clang-format"` That's build system agnostic.